File: Team_Workspace/dalia_ahmed/task2.py

gear ="D"
speed=0.0
def set_speedby_gear(gear):
    match gear:
        case "p":
            speed=0
            print("Parking")
        case "D":
            speed=10
            print("Driving Forward")
        case "R":
            speed=-5
            print("Reversing")
        case _:
            speed=0
            print("Error: Unknown Gear")
set_speedby_gear(gear)
        
          

File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task2/task2.cpp

//task2


#include <iostream>
using namespace std;

int main()
{
	char gear = 'D';
	double speed = 0.0;


	switch (gear) {
	case 'P':
		speed = 0;
		cout << "parking...";
		break;

	case 'D':
		speed = 10;
		cout << "Driving...";
		break;
	case 'R':
		speed = -5;
		cout << "Revdersing...";
		break;
	default:
		speed = 0;
		cout << "Error:Unknown Gear";


	}

}

